reporting-procedures.md

```diff
@@ -10,22 +10,16 @@ If you find that you need to make a report, and you cannot find the appropriate
 
 *General PSF reporting procedure:*
 
-The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias is listed at [LINK](). 
+The best way to contact the PSF Code of Conduct working group is by email at **<conduct@python.org>**. The members of the Code of Conduct Working Group who monitor this alias is listed at [PSF Code of Conduct Workgroup Charter](https://wiki.python.org/psf/ConductWG/Charter#List_of_Participants.2FWho_we_are). 
 
 In the event of a [conflict of interest](), you may directly contact any of the lead incident responders:
```

Comment:
should that link resolve somewhere?

Reply:
Maybe here? https://en.wikipedia.org/wiki/Conflict_of_interest

Reply:
The link is resolving for me with incognito mode so I'm not seeing this.

Reply:
The link goes one level up (when I'm at https://github.com/psf/community-code-of-conduct/blob/ee92c25ce2680ae99bd3437b33d73a8e6c105460/reporting-procedures.md it goes to https://github.com/psf/community-code-of-conduct/blob/ee92c25ce2680ae99bd3437b33d73a8e6c105460/ for example) for me.

Reply:
Right now GitHub resolves that `()` to `https://github.com/psf/community-code-of-conduct/blob/update-contacts` which is a 404